File: java-to-kotlin-starter-guide-master/src/main/java/com/lannstark/lec06/Lec06Main.java
package com.lannstark.lec06;

import java.util.Arrays;
import java.util.List;

public class Lec06Main {

  public static void main(String[] args) {

    //1
    final List<Long> numbers = Arrays.asList(1L, 2L, 3L);
    for (long number : numbers) {
        System.out.println(number);
    }

    //2-(1)
    for (int i = 1; i <= 5; i += 2) {
      System.out.println(i);
    }

    //2-(2)
    for (int i = 3; i >= 1; i --) {
      System.out.println(i);
    }

    //3
    int i = 1;
    while (i <= 3) {
      System.out.println(i);
      i++;
    }

  }

}

File: java-to-kotlin-starter-guide-master/src/main/java/com/lannstark/lec06/Lec06Main.kt
package com.lannstark.lec06

fun main() {
    //1.
    val numbers = listOf(1L, 2L, 3L)
    for (number in numbers) {
        println(number)
    }

    //2-(1)
    for (i in 1..5 step 2) {
        println(i)
    }

    //2-(2)
    for (i in 3 downTo 1) {
        println(i)
    }

    //3. while 문은 자바와 동일
    var i = 1
    while (i <= 3) {
        println(i)
        i++
    }
}
